deploy/helm/ifrcgo-helm/values-production.yaml

```diff
@@ -1,4 +1,26 @@
+environment: production
+
+env:
+  API_FQDN: goadmin.ifrc.org
+  AUTO_TRANSLATION_TRANSLATOR: lang.translation.IfrcTranslator
+  CACHE_MIDDLEWARE_SECONDS: "0"
+  CELERY_REDIS_URL: redis://ifrcgo-helm-redis:6379/1
+  DEBUG_EMAIL: ""
+  DJANGO_DB_NAME: postgres
+  DJANGO_DEBUG: ""
+  DOCKER_HOST_IP: ""
+  DJANGO_READ_ONLY: "false"
+  DJANGO_ADDITIONAL_ALLOWED_HOSTS: ""
+  ELASTIC_SEARCH_HOST: elasticsearch://ifrcgo-helm-elasticsearch:9200
+  FRONTEND_URL: go.ifrc.org
+  GO_ENVIRONMENT: production
+  IFRC_TRANSLATION_DOMAIN: https://microservices-staging.ifrc.org
+  SENTRY_DSN: https://342632f67ea942c9bde214c0bb7e91be@sentry.northeurope.cloudapp.azure.com/2
+  SENTRY_SAMPLE_RATE: "0.2"
+
 api:
+  domain:
+  additionalDomain:
```

Comment:
@batpad could you fill in these values?

Reply:
@sunu filled in `domain` and `additionalDomain` - let me know if you need me to fill in any others.